kong/plugins/ddtrace/handler.lua

```diff
@@ -182,30 +184,52 @@ local function apply_resource_name_rules(uri, rules)
     return table.concat(fragments)
 end
 
-local header_tags
+local function configure(conf)
+    local get_from_vault = kong.vault.get
+    local get_env = function(env_name)
+      local env_value, _ = get_from_vault(string.format("{vault://env/%s}", env_name))
+      return env_value
+    end
+
+    -- Build agent url
+    local agent_host = get_env("DD_AGENT_HOST") or conf.agent_host or "localhost"
+    local agent_port = get_env("DD_TRACE_AGENT_PORT") or conf.trace_agent_port or "8126"
+    if type(agent_port) ~= "string" then
+        agent_port = tostring(agent_port)
+    end
```

Comment:
`"8126"` is always a string. Can the user specify non-string values in `conf.trace_agent_port`? Can `kong.vault.get` return non-string values in response to `{vault://...}` queries?